servers/republik/modules/crowdfundings/graphql/resolvers/Membership.js

```diff
@@ -29,19 +29,25 @@ const membershipResolver = {
   },
 
   async overdue (membership, args, { pgdb }) {
-    if (!membership.active || !membership.latestPaymentFailedAt) {
+    if (!membership.active) return false
+
+    const periods = await membershipResolver.periods(
+      membership, null, { pgdb }
+    )
+    const hasPeriods = !!periods.length
+    if (!hasPeriods) {
+      console.trace(
+        // Sanity check: an active membership should always have a period.
+        `[data integrity] active membership without periode: ${membership.id}`
+      )
       return false
     }
 
-    const latest = await pgdb.public.membershipPeriods.findFirst({
-      membershipId: membership.id
-    }, { fields: '"endDate"', orderBy: ['endDate desc'] })
-    return !!(
-      membership.active &&
-      membership.latestPaymentFailedAt &&
-      membership.latestPaymentFailedAt > latest.endDate
-    )
+    const latestPeriod = periods[periods.length - 1]
```

Comment:
```suggestion
    const latestPeriod = periods[0]
```
periods is sorted `DESC`.